lib/active_merchant/billing/gateways/litle.rb

```diff
@@ -214,6 +214,11 @@ def add_payment_method(doc, payment_method)
             doc.cardholderAuthentication do
               doc.authenticationValue(payment_method.payment_cryptogram)
             end
+          elsif options[:order_source] && options[:order_source].start_with?('3ds')
```

Comment:
Are there other values for `order_source` that do not include 3DS related info? Just wondering if we need the additional `options[:order_source].start_with?('3ds')` conditional.